application-services/custom/camera-management/README.md

```diff
@@ -107,13 +108,25 @@ sudo apt install build-essential
 
    b. Under the `ports` section, find the entry for port 8554 and change the host_ip from `127.0.0.1` to either `0.0.0.0` or the ip address you put in the previous step.
 
-6. Run the following `make` command to run the edgex core services along with the Onvif and Usb device services. 
+6. Run the following `make` command to generate the edgex core services along with MQTT, Onvif and Usb device services.
 
   > **Note**: The `ds-onvif-camera` parameter can be omitted if no Onvif cameras are present, or the `ds-usb-camera` parameter can be omitted if no usb cameras are present.
 ```shell
-   make run no-secty ds-onvif-camera ds-usb-camera 
+   make gen no-secty ds-mqtt mqtt-broker ds-onvif-camera ds-usb-camera 
 ```   
 
+7. Configure [device-mqtt] service to send [Edge Video Analytics Microservice][evam] inference results into Edgex via MQTT
+
+      a. Copy complete [evam-mqtt-edgex](edge-video-analytics/evam-mqtt-edgex) folder into edgex-compose/compose-builder directory.
+
+      b. Copy and paste [docker-compose.override.yml](edge-video-analytics/evam-mqtt-edgex/docker-compose.override.yml) from the above copied folder into edgex-compose/compose-builder directory.
+         Insert full path of `edgex-compose/compose-builder` directory under volumes in this `docker-compose.override.yml`.
```

Comment:
I think it might be clearer to add another step and explain in more detail what/where they are inserting stuff. Like with an example of old -> new or something.

Reply:
Its been shown here https://github.com/EdgeX-Camera-Management/edgex-examples/blob/evam-edgex-docs/application-services/custom/camera-management/edge-video-analytics/evam-mqtt-edgex/docker-compose.override.yml#L28 what/where needs to be inserted.

Reply:
Ok. We may want to let the users know there is 2 locations it needs to be done at. the second one could be easy to miss.